undistro/zora
# UD-1521: Add fsgroup for persistent volume

## Description
Update the operator and download job so the trivy pods specify fsGroup.  This is needed when running as a non-root user, to guarantee access to volume mounts

## Linked Issues

## How has this been tested?
- Install 0.8.4 and upgrade to this version
- While upgrading check that the download job has fsGroup set to 0
- Check trivy cronjob has fsGroup set to 0
- Check trivy job runs to completion

## Checklist
